src/Cdn/docs/Test-AzFrontDoorCdnProfileCanMigrateOrNot.md

```diff
@@ -0,0 +1,145 @@
+---
+external help file:
+Module Name: Az.Cdn
+online version: https://docs.microsoft.com/powershell/module/az.cdn/test-azfrontdoorcdnprofilecanmigrateornot
+schema: 2.0.0
+---
+
+# Test-AzFrontDoorCdnProfileCanMigrateOrNot
```

Comment:
I think Test-AzFrontDoorCdnProfileMigration is simpler than Test-AzFrontDoorCdnProfileCanMigrateOrNot with same meaning, what's your thought?

Reply:
Yes, I agree and would fix this one.